api/src/test/java/org/openmrs/api/EncounterServiceTest.java

```diff
@@ -468,6 +469,20 @@ public void saveEncounter_shouldNotCascadeToExistingOrder() {
 		Boolean voided = (Boolean) Context.getAdministrationService().executeSQL(sql, true).get(0).get(0);
 		Assert.assertFalse(voided);
 	}
+
+	/**
+	 * @see EncounterService#hasDiagnosis(Encounter, Diagnosis)
+	 */
+	@Test
+	public void hasDiagnosis_shouldCheckIfEncounterHasDiagnosis(){
+		EncounterService encounterService = Context.getEncounterService();
+		DiagnosisService diagnosisService = Context.getDiagnosisService();
```

Comment:
This should go to the EncounterTest class

Reply:
@dkayiwa Kindly elaborate on why it should go to the EncounterTest class

Reply:
The convention is that when the changes are in the Encounter class, they are tested using EncounterTest

Reply:
Oh yes. Done